package.json

```diff
@@ -87,7 +88,7 @@
     "simple-eta": "^3.0.2"
   },
   "peerDependencies": {
-    "@nextcloud/vue": "^8.0.0-beta || ^8.0.0",
+    "@nextcloud/vue": "^8.11.0",
```

Comment:
Is this needed for the changes? Otherwise this might introduce duplicated dependencies for using apps.
Libraries should only pin the lowest supported version, the highest compatible version is picked by NPM on install and shared with the app.

Reply:
Not needed no.
We can keep `^8.0.0-beta || ^8.0.0`, it doesn't matter, `8.11.0` is installed in the lockfile

Reply:
Actually, it did matter.
I had to do this to update the nc/vue and see the latest changes on the `NcDialog`. Otherwise it was weirdly not working :thinking: 

Now that the lockfile is updated, I can revert I guess.
But this makes updating the dep complicated (check the auto dependabot updates, it [never offered](https://github.com/nextcloud-libraries/nextcloud-upload/pulls?q=is%3Apr+sort%3Aupdated-desc+is%3Aclosed+%22%40nextcloud%2Fvue%22) `@nextcloud/vue` since we changed that line)